src/functions/keycloak/list-users/handler.ts

```diff
@@ -0,0 +1,67 @@
+import { middyfy } from "@libs/lambda";
+import type { APIGatewayProxyHandler } from "aws-lambda";
+import { CreateKeycloakApiService, type KeycloakApiService } from "src/apis/keycloak-api-service";
+
+
+/**
+ * Response schema for lambda
+ */
+interface Response {
+    id: string
+    firstName: string
+    lastName: string
+    email: string
+    isActive: boolean
+    severaGuid: string
+    forecastId: number
+}
+
+/**
+ * Gets all users
+ * 
+ * @param api
+ * @returns Array of users 
+ */
+const listUsers = async (api: KeycloakApiService): Promise<Response[]> => {
+    try {
+        const users = await api.getUsers()
+
+        return users.map(user => {
+            return {
+                id: user.id,
+                firstName: user.firstName,
+                lastName: user.lastName,
+                email: user.email,
+                isActive: user.isActive,
+                severaGuid: user.severaGuid,
+                forecastId: user.forecastId
+            }
+    })
+    } catch (error) {
+        throw error("listUsers(): Error when listing users")
+    }
+}
+
+/**
+ * Lambda for listing users
+ * 
+ * @param _event event
+ */
+const listUsersHandler: APIGatewayProxyHandler = async (_event) => {
+    try {
+        const api = CreateKeycloakApiService();
+        const users = await listUsers(api)
```

Comment:
Similar to the find users I see no need for a separate listUsers function here as this function is small enough. Just call the api.listUsers from here and handle appropriately